ironfish/src/wallet/account.ts

```diff
@@ -458,7 +458,7 @@ export class Account {
       }
     }
 
-    for (const noteHash of this.nonChainNoteHashes.values()) {
+    for (const noteHash of this.nonChainNoteHashes.toKeys()) {
```

Comment:
This is an existing bug in the typedef files I noticed. I wondered why this doesn't use the iterator. It looks like the typedef file in our repo doesn't match the actual javascript library.

 - https://github.com/iron-fish/ironfish/blob/master/ironfish/src/typedefs/buffer-map.d.ts#L38
 - https://github.com/chjj/buffer-map/blob/master/lib/buffer-map.js#L151-L153